### ISSUE TITLE
missing field `totalSize`

### ISSUE THREAD

**Issue by pgilad** [jovobe/alfred-confluence-workflow#1]
When running the code I get:

```
Error: error decoding response body: missing field `totalSize` at line 1 column 2223

Caused by:
    missing field `totalSize` at line 1 column 2223

Process finished with exit code 1
```

And I think `total_size` field of the `ApiResponse` should be wrapped as `Option`

**Comment by jovobe** [comment]
@pgilad which version of Confluence are you using?

**Comment by pgilad** [comment]
<img width="603" alt="image" src="https://user-images.githubusercontent.com/4533329/161485133-dd7ded6b-538c-4e2c-b8be-f82c328c9ece.png">
Does that make sense?

**Comment by jovobe** [comment]
Absolutely, thanks @pgilad! This should be fixed in db4969d7572452dfaeb6d206803cfca11b1b4ac7. Please comment to this issue if the bug still exists.
